jakeruss-capecoral.md

```diff
@@ -0,0 +1,8 @@
+# Cape Coral, FL
+
+### Restaurants
+
+### Tourist Attractions
+
+### Local Attractions
+- Cayo Costa State Park
```

Comment:
@JakeRuss is this an all-day park? Is there enough stuff to do to take up your entire day?